set.mm

```diff
@@ -610,7 +610,7 @@ that the result (using the rules described in ~ wi and ~ wn ) always
     $( Rule of Modus Ponens.  The postulated inference rule of propositional
        calculus.  See e.g.  Rule 1 of [Hamilton] p. 73.  The rule says, "if
        ` ph ` is true, and ` ph ` implies ` ps ` , then ` ps ` must also be
-       true."  This rule is sometimes called "detachment," since it detaches
+       true".  This rule is sometimes called "detachment," since it detaches
```

Comment:
Suggest to change a comment consistently to logical style.  Here 'detachment', 'modus ponendo ponens'  need transition, too.

Reply:
You're right. Fixed. I took the opportunity to add one more commit to standardize more spellings. That's the last in this PR.